### ISSUE TITLE
Pixel H265 video: Device freezes and reboots on seek

### ISSUE THREAD

**Issue by ian-wevr** [google/ExoPlayer#2587]
### Issue description
In Exoplayer demo app, device freezes and then reboots after seeking in either direction during playback of a video-only mp4 containing a single h.265 stream (see test content section). Happens regardless of if playback is paused or unpaused.

### Reproduction steps
Added entry to demo app's media.exolist.json file:

```
{
    "name": "Test",
    "samples": [
      {
        "name": "Test",
        "uri" : "file:///storage/self/primary/fasttest.mp4"
      },
}
```

Launch demo, select the test entry. Video playback will appear as expected -- test content does not have a moving image. Seek as desired, device freezes and reboot occurs.

### Link to test content
https://drive.google.com/file/d/0B3bMZi3-ksViUXJ3azRRLWhKXzQ
(should be publicly viewable)

### Version of ExoPlayer being used
2.3.0

### Device(s) and version(s) of Android being used
Pixel XL, Android 7.1.1, reproduces 100%
Pixel, Android 7.1.1, reproduces 100%
Moto Z Droid, Android 7.0, never reproduces
Samsung Galaxy S7 edge (SM-G935F), Android 6.0.1, never reproduces

### A full bug report captured from the device
Pixel XL bug report: https://drive.google.com/file/d/0B3bMZi3-ksViQ2FRbl9pQ3h3SVE
Pixel bug report: https://drive.google.com/file/d/0B3bMZi3-ksViTUpxVlV1VDByNGs

**Comment by ojw28** [comment]
This is a device specific issue. My understanding is that Qualcomm are investigating.
[Internal bug ref: 36511618].

**Comment by ian-wevr** [comment]
@ojw28 Thanks for the info!

> My understanding is that Qualcomm are investigating.

Is there another issue tracker or site where I should/can continue to be involved or track the status of the underlying issue?

**Comment by ojw28** [comment]
No; we'll update this issue when possible.

**Comment by kneep** [comment]
I saw a similar issue on Pixel XL when MX Player was playing an hevc 10-bit video. The media scanner even did not parse the video so it did not show in the media database. When I found the video in a file explorer and played it with MX Player and sought back and forth, the handset stuck for a while and rebooted.  I tried the same thing on a ZTE Axon 7 which is also built on msm8996, the media scanner could recognize the video and it played smoothly with hardware acceleration. Seems like google removed some codec software of Qualcomm from Pixel XL?
